doc/install.rst

```diff
@@ -269,26 +269,33 @@ python library for Windows, Mac OSX and Linux.
 Anaconda offers scikit-learn as part of its free distribution.
 
 
-Intel conda channel
+Intel Extension for Scikit-learn
 -------------------
 
-Intel maintains a dedicated conda channel that ships scikit-learn:
+Intel maintains an optimized x86 package, avaialbe in PyPi, conda main 
+and conda-forge:
```

Comment:
I'm assuming this means "conda defaults channel", but I suspect that saying "conda" is enough:

```suggestion
Intel maintains an optimized x86 package, avaialbe in PyPi, conda 
and conda-forge:
```